dashboard/app/models/objective.rb

```diff
@@ -7,9 +7,11 @@
 #  lesson_id  :integer
 #  created_at :datetime         not null
 #  updated_at :datetime         not null
+#  key        :string(255)
```

Comment:
I know this will break current objective but do we want to make this `not null`?

Reply:
yes, we do... what do you think, should I do that now? because Objectives are not in fixtures, this should only affect our immediate dev team. we should be able to unblock ourselves via `Objective.destroy_all!` and then repeating the import.

Reply:
I think I'd bundle it with the PR that adds a way to set a key (unless I missed it in this PR). Especially with staging closing so frequently I'm worried we'll end up in a spot where we all have to work off a branch in order to get any work done

Reply:
That makes sense.  I can do it in a separate PR which updates tests and makes sure not to break anything.

Reply:
Sounds good!